fire/inspectutils.py

```diff
@@ -72,11 +72,18 @@ def GetArgSpec(fn):
   fn, skip_arg = _GetArgSpecFnInfo(fn)
 
   try:
-    argspec = inspect.getargspec(fn)
+
+    if six.PY2:
+      argspec = inspect.getargspec(fn)
+      keywords = argspec.keywords
+    elif six.PY3:
```

Comment:
nit: Let's change this to "else:" for consistency with other PY2 checks, and so that a reader/static analyzer can guarantee that one of the two branches will be taken.